velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -807,6 +807,55 @@ struct MakeYMIntervalFunction {
   }
 };
 
+template <typename T>
+struct MakeDTIntervalFunction {
```

Comment:
Could you please add the link of corresponding Spark's implementation to the PR description? And also remember to update the doc.

Reply:
I think https://github.com/apache/spark/blob/branch-3.5/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/intervalExpressions.scala#L391 is the Spark's implementation for this function.